system_tests/logging_.py

```diff
@@ -193,3 +197,18 @@ def test_reload_sink(self):
         sink.reload()
         self.assertEqual(sink.filter_, DEFAULT_FILTER)
         self.assertEqual(sink.destination, uri)
+
+    def test_update_sink(self):
+        bucket_uri = self._init_storage_bucket()
+        dataset_uri = self._init_bigquery_dataset()
+        UPDATED_FILTER = 'logName:syslog'
```

Comment:
It is an arbitrary filter expression, limiting entries exported via the sink to those whose `logName` field matches that string.  We use a narrower filter (`logName:syslog AND severity:ERROR`) as the default filter.